.circleci/cimodel/data/binary_build_data.py

```diff
@@ -51,13 +51,16 @@ def get_processor_arch_name(cuda_version):
         ],
     )),
     # Skip CUDA-9.2 builds on Windows
-    windows=([v for v in dimensions.CUDA_VERSIONS if v != '92'], OrderedDict(
-        wheel=dimensions.STANDARD_PYTHON_VERSIONS,
-        conda=dimensions.STANDARD_PYTHON_VERSIONS,
-        libtorch=[
-            "3.7",
-        ],
-    )),
+    windows=(
+        [v for v in dimensions.CUDA_VERSIONS if v not in ['92', '110']],
```

Comment:
Disabling CUDA 11 builds here for windows temporarily to unblock linux builds

Reply:
Hey, please cc me or the windows team about it next time.